test/blackbox-tests/test-cases/dune-init.t/run.t

```diff
@@ -251,24 +251,23 @@ Comments in dune files are preserved
 Will not create components with invalid names
 
   $ dune init lib invalid-component-name ./_test_lib
-  dune init: NAME argument: invalid component name
-             `invalid-component-name'
-             Library names must be non-empty and composed only of the
-             following
-             characters: 'A'..'Z', 'a'..'z', '_' or '0'..'9'.
-  Usage: dune init [OPTION]... COMPONENT NAME [PATH]
-  Try `dune init --help' or `dune --help' for more information.
+  dune: NAME argument: invalid component name `invalid-component-name'
+        Library names must be non-empty and composed only of the
+        following
+        characters: 'A'..'Z', 'a'..'z', '_' or '0'..'9'.
+  Usage: dune init [OPTION]… COMPONENT NAME [PATH]
+  Try 'dune init --help' or 'dune --help' for more information.
   [1]
   $ test -f ./_test_lib
   [1]
 
 Will fail and inform user when invalid component command is given
 
   $ dune init foo blah
-  dune init: COMPONENT argument: invalid value `foo', expected one of
-             `executable', `library', `project' or `test'
-  Usage: dune init [OPTION]... COMPONENT NAME [PATH]
-  Try `dune init --help' or `dune --help' for more information.
+  dune: COMPONENT argument: invalid value 'foo', expected one of 'executable',
+        'library', 'project' or 'test'
+  Usage: dune init [OPTION]… COMPONENT NAME [PATH]
+  Try 'dune init --help' or 'dune --help' for more information.
   [1]
 
 Will fail and inform user when an invalid option is given to a component
```

Comment:
```suggestion
This command will fail and inform user when an invalid option is given to a component
```

Reply:
This is better but unrelated to this PR so I'd rather leave it off the diff.